### PR TITLE
NVDA is not reading the name of 'Expand or collapse all groups' column

### PR DESCRIPTION
**Pull request by gggjtgxing**
## Previous Behavior
NVDA is not reading the name of 'Expand or collapse all groups' when you focus on the expand/collapse button
image

Same issue does not repro with narrator.

## Related Issue(s)
https://onedrive.visualstudio.com/STS/_workitems/edit/1380388

* Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/294d8a773e2d154f04addf24ff8600a7bb408a31'>294d8a773e2d154f04addf24ff8600a7bb408a31</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/FocusZone-mount_base.html">1935</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/FocusZone-mount.html">1924</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/FocusZone-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/BaseButton-mount_base.html">1193</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/BaseButton-mount.html">1200</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Breadcrumb-mount_base.html">2787</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Breadcrumb-mount.html">2845</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Checkbox-mount_base.html">2649</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Checkbox-mount.html">2621</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/CheckboxBase-mount_base.html">2376</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/CheckboxBase-mount.html">2352</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ChoiceGroup-mount_base.html">4294</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ChoiceGroup-mount.html">4299</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ComboBox-mount_base.html">1194</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ComboBox-mount.html">1178</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/CommandBar-mount_base.html">9410</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/CommandBar-mount.html">9433</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ContextualMenu-mount_base.html">10592</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ContextualMenu-mount.html">10810</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DefaultButton-mount_base.html">1362</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DefaultButton-mount.html">1376</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DetailsRow-mount_base.html">3404</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DetailsRow-mount.html">3371</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DetailsRowFast-mount_base.html">3351</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DetailsRowFast-mount.html">3376</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DetailsRowNoStyles-mount_base.html">3255</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DetailsRowNoStyles-mount.html">3248</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Dialog-mount_base.html">3000</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Dialog-mount.html">2976</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DocumentCardTitle-mount_base.html">614</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/DocumentCardTitle-mount.html">576</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Dropdown-mount_base.html">3108</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Dropdown-mount.html">3147</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/FocusTrapZone-mount_base.html">1967</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/FocusTrapZone-mount.html">1996</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/FocusZone-mount_base.html">1935</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/FocusZone-mount.html">1924</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/FocusZone-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedList-mount_base.html">1839</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedList-mount.html">2049</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedList-virtual-rerender_base.html">1117</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedList-virtual-rerender.html">1099</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1619</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedList-virtual-rerender-with-unmount.html">1621</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedListV2-mount_base.html">561</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedListV2-mount.html">567</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedListV2-virtual-rerender_base.html">540</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedListV2-virtual-rerender.html">558</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">571</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">567</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/IconButton-mount_base.html">1822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/IconButton-mount.html">1810</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Label-mount_base.html">752</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Label-mount.html">762</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Layer-mount_base.html">4166</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Layer-mount.html">4175</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Link-mount_base.html">846</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Link-mount.html">872</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/MenuButton-mount_base.html">1603</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/MenuButton-mount.html">1610</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/MessageBar-mount_base.html">2343</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/MessageBar-mount.html">2347</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Nav-mount_base.html">3088</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Nav-mount.html">3094</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/OverflowSet-mount_base.html">1425</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/OverflowSet-mount.html">1420</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Panel-mount_base.html">2522</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Panel-mount.html">2558</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Persona-mount_base.html">1268</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Persona-mount.html">1296</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Pivot-mount_base.html">1540</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Pivot-mount.html">1534</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/PrimaryButton-mount_base.html">1476</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/PrimaryButton-mount.html">1498</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Rating-mount_base.html">6934</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Rating-mount.html">6958</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/SearchBox-mount_base.html">1524</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/SearchBox-mount.html">1498</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Shimmer-mount_base.html">2925</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Shimmer-mount.html">2892</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Slider-mount_base.html">2129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Slider-mount.html">2095</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/SpinButton-mount_base.html">4336</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/SpinButton-mount.html">4237</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Spinner-mount_base.html">853</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Spinner-mount.html">844</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/SplitButton-mount_base.html">2867</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/SplitButton-mount.html">2823</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Stack-mount_base.html">887</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Stack-mount.html">870</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/StackWithIntrinsicChildren-mount_base.html">2245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/StackWithIntrinsicChildren-mount.html">2245</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/StackWithTextChildren-mount_base.html">4939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/StackWithTextChildren-mount.html">4936</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/SwatchColorPicker-mount_base.html">9530</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/SwatchColorPicker-mount.html">9518</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/TagPicker-mount_base.html">2422</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/TagPicker-mount.html">2376</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/TeachingBubble-mount_base.html">80150</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/TeachingBubble-mount.html">80327</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Text-mount_base.html">834</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Text-mount.html">813</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/TextField-mount_base.html">1549</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/TextField-mount.html">1551</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ThemeProvider-mount_base.html">1439</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ThemeProvider-mount.html">1456</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ThemeProvider-virtual-rerender_base.html">1131</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ThemeProvider-virtual-rerender.html">1149</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2013</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Toggle-mount_base.html">1136</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/Toggle-mount.html">1132</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/buttonNative-mount_base.html">535</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26443/perf-test/buttonNative-mount.html">536</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react/src/components/DetailsList/DetailsHeader.styles.ts [hunk 1]
```diff
@@ -181,6 +181,10 @@ export const getDetailsHeaderStyles = (props: IDetailsHeaderStyleProps): IDetail
       },
     ],
 
+    groupExpanderContainer: {
```
Comment (ThomasMichon) [inline packages/react/src/components/DetailsList/DetailsHeader.styles.ts:184]: If we use `display: contents`, don't we risk the screen readers ignoring the element in the accessibility tree? I thought that at this point, the only thing the parent element can do is define CSS variables.

**Review by ThomasMichon — APPROVED** [review]

File: packages/react/src/components/DetailsList/DetailsHeader.base.tsx [hunk 2]
```diff
@@ -280,20 +280,21 @@ export class DetailsHeaderBase
             ]
```
Comment (fabricteam) [inline packages/react/src/components/DetailsList/DetailsHeader.base.tsx:280]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26443&commitId=f1a517db0eb73344f5c15eb6a00882891f88bc27&env=prod)

✅ There was 4 screenshots added, 0 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Pull request closed without merging**